examples/angular-cli/e2e/app.e2e-spec.ts

```diff
@@ -0,0 +1,15 @@
+import { AppPage } from './app.po';
```

Comment:
These are tests generated by the angular-cli right?

Reply:
Yeah, that's right